hack/kind-cluster-build.sh

```diff
@@ -94,6 +94,16 @@ KUBECTL_BIN=${OUTPUT_BIN}/kubectl
 HELM_BIN=${OUTPUT_BIN}/helm
 KIND_BIN=${OUTPUT_BIN}/kind
 
+# create registry container unless it already exists
+reg_name='registry'
```

Comment:
I'm not sure if we want to accept arguments for setting the registry name and port. Any thoughts?

Reply:
@namco1992 I think it might be better to use parameters here. For example, the registry does not use `5000`

Reply:
@cwen0 sure I will work on it.